plugins/source/azure/resources/services/cdn/profiles.go

```diff
@@ -1,1316 +1,102 @@
-				Description: "SecurityPolicy securityPolicy association for AzureFrontDoor profile",
-				Resolver:    fetchCdnProfileSecurityPolicies,
-				Columns: []schema.Column{
-					{
-						Name:        "profile_cq_id",
-						Description: "Unique CloudQuery ID of azure_cdn_profiles table (FK)",
-						Type:        schema.TypeUUID,
-						Resolver:    schema.ParentIdResolver,
-					},
-					{
-						Name:        "parameters",
-						Description: "object which contains security policy parameters",
-						Type:        schema.TypeJSON,
-						Resolver:    schema.PathResolver("SecurityPolicyProperties.Parameters"),
-					},
-					{
-						Name:        "provisioning_state",
-						Description: "Provisioning status",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SecurityPolicyProperties.ProvisioningState"),
-					},
-					{
-						Name:        "deployment_status",
-						Description: "Possible values include: 'DeploymentStatusNotStarted', 'DeploymentStatusInProgress', 'DeploymentStatusSucceeded', 'DeploymentStatusFailed'",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SecurityPolicyProperties.DeploymentStatus"),
-					},
-					{
-						Name:        "id",
-						Description: "Resource ID",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("ID"),
-					},
-					{
-						Name:        "name",
-						Description: "Resource name",
-						Type:        schema.TypeString,
-					},
-					{
-						Name:        "type",
-						Description: "Resource type",
-						Type:        schema.TypeString,
-					},
-					{
-						Name:        "created_by",
-						Description: "An identifier for the identity that created the resource",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SystemData.CreatedBy"),
-					},
-					{
-						Name:        "created_by_type",
-						Description: "The type of identity that created the resource",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SystemData.CreatedByType"),
-					},
-					{
-						Name:     "created_at_time",
-						Type:     schema.TypeTimestamp,
-						Resolver: schema.PathResolver("SystemData.CreatedAt.Time"),
-					},
-					{
-						Name:        "last_modified_by",
-						Description: "An identifier for the identity that last modified the resource",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SystemData.LastModifiedBy"),
-					},
-					{
-						Name:        "last_modified_by_type",
-						Description: "The type of identity that last modified the resource",
-						Type:        schema.TypeString,
-						Resolver:    schema.PathResolver("SystemData.LastModifiedByType"),
-					},
-					{
-						Name:     "last_modified_at_time",
-						Type:     schema.TypeTimestamp,
-						Resolver: schema.PathResolver("SystemData.LastModifiedAt.Time"),
-					},
-				},
-			},
+			endpoints(), ruleSets(), securityPolicies(),
```

Comment:
The relations are now extracted to separate files like plugins/source/azure/resources/services/cdn/endpoints.go